src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -69,6 +69,9 @@ class GcsServer {
   virtual std::unique_ptr<rpc::WorkerInfoHandler> InitWorkerInfoHandler();
 
  private:
+  /// Set gcs server address to redis.
```

Comment:
would you add some details on why this is required? e.g. to let clients to get new gcs server ip address if it crashes ...

Reply:
fixed

Reply:
fixed